asf_theme/partials/footer.html

```diff
@@ -115,8 +115,10 @@ <h1>Contact Info</h1>
       <div class="md-footer-copyright">
         <div class="md-footer-copyright__highlight">
           {% if config.copyright or config.theme.copyright %}
-            &copy; <script type="text/javascript">
-                document.write(new Date().getFullYear());
+            &copy; <span id="date"><span>
+            <script type="text/javascript">
+                d = new Date().getFullYear();
+                document.getElementById(date).innerHTML=d;
```

Comment:
`getElementById("date")` need quotes.  And `copyright-date` would be a better, more explicit element ID

Reply:
and something like `current_year` would be better than `d`